runtime/doc/vim_diff.txt

```diff
@@ -28,6 +28,10 @@ these differences.
 ==============================================================================
 2. Option defaults					  *nvim-option-defaults*
 
+- syntax highlighting is enabled by default (can be disabled with the "-u NONE"
+  cmdline argument)
+- filetype-related plugins and scripts are enabled by default (can be disabled
+  with the "-u NONE" cmdline argument)
```

Comment:
oh, and I think Vim already has a concept of a command line (`:`), but I wonder what other wording could be used.